src/renderer/components/+catalog/catalog-entity.store.ts

```diff
@@ -106,6 +106,14 @@ export class CatalogEntityStore extends ItemStore<CatalogEntityItem> {
   }
 
   loadAll() {
+    if (this.activeCategory) {
+      this.activeCategory.emit("onLoad");
```

Comment:
This reads weird when used, because the consumer would do `.on("onLoad", <listener>)`. I think it should be just `"load"`.

Reply:
I agree